src/guiguts/misc_tools.py

```diff
@@ -1224,3 +1230,111 @@ def asterisk_check() -> None:
             line, IndexRange(match.rowcol, end_rowcol), match.rowcol.col, end_rowcol.col
         )
     checker_dialog.display_entries()
+
```

Comment:
Extra blank line?

Reply:
No - the code prettifier forces 2 blank lines before the start of a new class (TextMarkupConvertorDialog), and 1 line between functions.